src/Workspaces/Core/Portable/Editing/GenerationOptions.cs

```diff
@@ -1,21 +1,28 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CodeStyle;
 using Microsoft.CodeAnalysis.Options;
 
 namespace Microsoft.CodeAnalysis.Editing
 {
     internal class GenerationOptions
     {
         public static readonly PerLanguageOption<bool> PlaceSystemNamespaceFirst = new PerLanguageOption<bool>(nameof(GenerationOptions),
+            CodeStyleOptionGroups.Usings,
```

Comment:
note: we've generally called this 'imports' rather than 'usings' as 'imports' was found to at least make some amount of sense for both C# and VB, whereas 'usings' was completely non-sensical for the VB crowd.

Reply:
📝 This was not changed by this pull request.

@dpoeschl should this be in scope for this pull request?